R/aggregation.R

```diff
@@ -93,12 +94,18 @@ get_aggregation <- function(streamid,
                 })
             rm(req)
             rm(response)
+            if (name_streams) {
```

Comment:
It might not be necessary to name a stream and show warning message if you just retrieve a single stream. As it will be easy to rename column name outside rsenaps. 
It would be better to return consistent column names from rsenaps and let users to decide how to rename it.